ilastik/applets/edgeTrainingWithMulticut/edgeTrainingWithMulticutGui.py

```diff
@@ -39,7 +43,57 @@ def initAppletDrawerUi(self):
         multicut_required_slots = (op.Superpixels, op.Rag, op.EdgeProbabilities, op.EdgeProbabilitiesDict)
         self.__cleanup_fns.append(guiutil.enable_when_ready(multicut_box, multicut_required_slots))
 
+        def _handle_easy_predict_button_clicked():
+            self.training = not self.training
+            training_controls.setEnabled(self.training)
+            multicut_controls.setEnabled(self.training)
+            op.opEdgeTraining.opPredictEdgeProbabilities.TrainRandomForest.setValue(self.training)
+            op.FreezeClassifier.setValue(False) 
+
+            def updateThread():
+                """
+                Copied over from code for Live Multicut button in:
+   
+                ilastik.applets.multicut.multicutGui._handle_multicut_update_clicked
+
+                The need for threading as explained there:
+                '''
+                This is hacky, but for now it's the only way to do it. We need to 
+                make sure the rendering thread has actually seen that the cache
+                has been updated before we ask it to wait for all views to be 100% 
+                rendered. If we don't wait, it might complete too soon (with the 
+                old data).
+                '''
+                """
+                with self.set_updating():
+                    self.topLevelOperatorView.FreezeCache.setValue(False)
+                    ndim = len(self.topLevelOperatorView.Output.meta.shape)
+                    self.topLevelOperatorView.Output((0,) * ndim, (1,) * ndim).wait()
+     
+                    # Wait for the image to be rendered into all three image views
+                    for imgView in self.editor.imageViews:
+                        if imgView.isVisible():
+                            imgView.scene().joinRenderingAllTiles()
+                    self.topLevelOperatorView.FreezeCache.setValue(True)
+     
+            self.getLayerByName("Multicut Edges").visible = True
+            # self.getLayerByName("Multicut Segmentation").visible = True
+            th = threading.Thread(target=updateThread)
+            th.start()
+
+            op.FreezeClassifier.setValue(False) 
```

Comment:
I had to unfreeze the classifier in order to get the 'Easy Predict' button to update the view with multicut edges. I am not completely sure why this is the case when the classifier should not be used for this computation, and there might be a more elegant solution. Also if I freeze the classifier here, ilastik stops responding when 'Easy Predict' is toggled.